target/linux/ramips/dts/mt7621_sercomm_s3.dtsi

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	aliases {
+		led-boot = &led_rgb_status_green;
+		led-failsafe = &led_rgb_status_red;
+		led-running = &led_rgb_status_green;
+		led-upgrade = &led_rgb_status_red;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		/*
+		 * Two RGB LEDs are connected as one, OpenWrt does not yet support
+		 * multicolor LEDs.
+		 */
+
+		led_rgb_status_red: led-0 {
+			label = "red:status";
+			color = <LED_COLOR_ID_RED>;
+			function = LED_FUNCTION_PANIC;
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+			panic-indicator;
+		};
+
+		led_rgb_status_green: led-1 {
+			label = "green:status";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 15 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_rgb_status_blue: led-2 {
+			label = "blue:wan";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_WAN;
+			gpios = <&gpio 13 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "WPS Button";
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		reset {
+			label = "Reset Button";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&gmac1 {
+	status = "okay";
+	label = "wan";
+	phy-handle = <&ethphy0>;
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie0 { // MT7615N
+	wlan_5g: wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0 0 0 0 0>;
+		ieee80211-freq-limit = <5000000 6000000>;
+	};
+};
+
+&pcie1 { // MT7603EN
```

Comment:
I would suggest use MAC address scheme (here and for 5g) from the newest firmware v.2.09 where 2g MAC is not equal to the LAN MAC.

Reply:
In the OEM firmware v.1.0.12, this was exactly the case. Does it make sense?

Reply:
> Does it make sense?

Makes sense, since the vendor has fixed it in new firmware versions.
"Network nodes with multiple network interfaces, such as routers and multilayer switches, must have a unique MAC address for each NIC in the same network."

Reply:
But then it turns out that the manufacturer in vain indicated the MAC addresses of the wireless interfaces in their calibrations?

Reply:
> But then it turns out that the manufacturer in vain indicated the MAC addresses of the wireless interfaces in their calibrations?

Manufacturer made a right decision not to use them in new fw releases because they aren't correct.

Reply:
Added to commit)